src/lib/types/experimental.ts

```diff
@@ -48,6 +48,7 @@ export type IFlagKey =
     | 'createdByUserIdDataMigration'
     | 'showInactiveUsers'
     | 'inMemoryScheduledChangeRequests'
+    | 'trafficDataUsage'
```

Comment:
Maybe "collect traffic data" or "collect traffic data in unleash"?

Reply:
It's also going to control the UI for data usage, does that impact how you feel about the name?

Reply:
as discussed I've changed it to `collectTrafficDataUsage`